lookout/core/bytes_to_unicode_converter.py

```diff
@@ -82,15 +82,14 @@ def _convert_position(self, byte_position: bblfsh.Position) -> bblfsh.Position:
         offset = self._byte_to_str_offset[byte_position.offset]
         line_num = numpy.argmax(self._lines_offset > offset) - 1
         col = offset - self._lines_offset[line_num]
+        # line number can be changed. File example:
+        # github.com/src-d/style-analyzer/blob/ed9324d783eb0082d2f59de336190c8805a33c75/lookout/style/format/tests/bugs/002_bad_line_positions/browser-policy-content.js
         line = self._lines[line_num]
         if len(line) == col:
             if line.splitlines()[0] != line:
                 # ends with newline
                 line_num += 1
                 col = 0
-        assert line_num + 1 == byte_position.line, \
```

Comment:
I guess that it may be bigger, but never smaller. Shall we change this assertion to check `>=`?

Reply:
let's try and see :)

Reply:
At least test in style-analyzer is passed.